components/media-feature/Component.jsx

```diff
@@ -2,9 +2,10 @@ import React from "react";
 
 // import specific templates for the component
 import { Container } from "../../packages/grids/Container";
-import { SidebarHeading } from "../../packages/headings/Heading";
```

Comment:
I think we still need this. See comment below.

Reply:
Done.